src/pages/404.js

```diff
@@ -1,12 +1,57 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+import { Grid, Typography, Button, withStyles } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
 
 import Layout from '../components/layouts/Layout';
+import { spacing } from '../components/styledComponents';
+import { getPageUrl } from '../Routes';
 
-const NotFoundPage = () => (
+const NotFoundPage = ({ classes: { container, image } }) => (
   <Layout routeSlug="NotFound">
-    <h1>NOT FOUND</h1>
-    <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
+    <Grid className={container} container direction="column" alignItems="center">
+      <Grid item>
+        <Typography variant="h1" color="secondary">
+          Oops!
+          <br />
+          Page Not Found
+        </Typography>
+        <Typography variant="subtitle1" color="secondary">
+          The page you are looking for doesn&apos;t exit or an error occurred!
+        </Typography>
+      </Grid>
+      <Grid item>
+        <img className={image} src="/404.webp" alt="404 Not Found" />
```

Comment:
Is this really worth using `.webp`? How big would this file be in regular `jpg`?

Reply:
@bencpeters 

As the image is animated, I couldn't use `jpg`

I use `webp` because it is the lightest format I found for animated images

For this image
- `.webp` --> `406KB`
- `.gif` --> `1.3MB`
- `.png` --> `2.2MB`

`.gif` looks like a good middle

Thought?

Reply:
`webp` is fine in that case.